fs/dafsstream/dafsstream.cpp

```diff
@@ -1471,6 +1471,11 @@ IRowWriter *createRowWriter(IDFUFilePartWriter *partWriter)
             serializer.setown(meta->createDiskSerializer(nullptr, 1));
         }
         virtual void putRow(const void *row) override
```

Comment:
Is this ever called at the moment?  If so the caller will need to be changed, but so far I haven't found where createRowWriter() is called from.

Reply:
I ran gtags against our source and then asked VSCode to find all references to that version of that function. It did not find any (but did find 7 calls to other same-named functions).

Reply:
I also renamed that function and verified that the platform compiled successfully. Could it be called from generated code?